dev/data/dashboard.json

```diff
@@ -1604,8 +1604,8 @@
               {
                 "x": 0,
                 "y": 0,
-                "width": 12,
-                "height": 6,
+                "width": 6,
+                "height": 4,
```

Comment:
Modified to values different than the defaults to get more signal out of the e2e test that validates duplicates keep the dimensions of the original panel.